src/hardware/pounder/mod.rs

```diff
@@ -40,10 +47,10 @@ pub enum Error {
 #[derive(Debug, Copy, Clone)]
 #[allow(dead_code)]
 pub enum Channel {
-    In0,
-    In1,
-    Out0,
-    Out1,
+    In0 = 0,
```

Comment:
What do these values correspond to? Would be worth documenting. It's not clear to me from reading the changes

Reply:
It was in the commit message to 3616f1fa. Will ad a comment.